contents/navigation/navbars/navbar-with-search/index.tsx

```diff
@@ -0,0 +1,180 @@
+      { icon: "✨", label: "Reports" },
+      { icon: "🛒", label: "Orders" },
+      { icon: "📅", label: "Events" },
+      { icon: "🙈", label: "Debts" },
+      { icon: "💁‍♀️", label: "Customers" },
+    ],
+    general: [
+      { icon: ActivityIcon, label: "Activity", number: "3" },
+      { icon: InfoIcon, label: "Tasks", number: "4" },
+      { icon: UserIcon, label: "Contacts", number: "" },
+    ],
+  }
+
+  return (
+    <VStack
+      as="nav"
+      borderRight="1px solid"
+      borderRightColor="border"
+      fontFamily="-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif"
+      h="full"
+      minH="700px"
+      w="300px"
+    >
+      <LinkBox
+        as="article"
+        cursor="pointer"
+        maxW="sm"
+        transition="all 0.1s"
+        _hover={{ bg: "#F5F5F5" }}
+      >
+        <LinkOverlay>
+          <Flex align="center" gap="md" p="md">
+            <Avatar
+              name="Hirotomo Yamada"
+              src="https://avatars.githubusercontent.com/u/84060430?v=4"
+            />
+            <Flex align="center" gap="lg">
+              <Box>
+                <Text as="b" fontSize="sm">
+                  Hirotomo Yamada
+                </Text>
+                <Text color="gray.400" fontSize="xs">
+                  hirotomo.yamada@avap.co.jp
+                </Text>
+              </Box>
+              <ChevronRightIcon />
+            </Flex>
+          </Flex>
+        </LinkOverlay>
+        <Divider variant="solid" />
+      </LinkBox>
+
+      <VStack as="nav" px="md">
+        <InputGroup size="sm">
+          <InputLeftElement>
+            <Search color="gray.500" />
+          </InputLeftElement>
+          <Input placeholder="Search" pr="4.6rem" />
+          <InputRightElement w="4.6rem">
+            <Text color="#080808">
+              <Kbd>Ctrl + K</Kbd>
+            </Text>
+          </InputRightElement>
+        </InputGroup>
+      </VStack>
+
+      <Flex direction="column" px="1">
+        {tabs.general.map((tab) => (
+          <Box key={tab.label}>
+            <LinkBox
+              as="article"
+              cursor="pointer"
+              maxW="sm"
+              transition="all 0.1s"
+              _hover={{ bg: "#F5F5F5" }}
+            >
+              <LinkOverlay>
+                <Flex align="center" gap="md" px="4" py="2">
+                  <tab.icon color="#909090" />
+                  <Text color="#505050" fontSize="sm">
+                    {tab.label}
+                  </Text>
+                  <Spacer />
+                  {tab.number ? (
+                    <Indicator
+                      label={tab.number}
+                      offset={1.5}
+                      ping
+                      pingScale={1.4}
+                      placement="right"
+                    >
+                      <Avatar display="none" />
+                    </Indicator>
```

Comment:
I am not sure if Yamada UI provides a dedicated component for displaying a number indicator, so I did a workaround to display the number by setting the Avatar component's `display` attribute to `none` to achieve the expected result